### ISSUE TITLE
Add option to not fail build if nothing is committed

### ISSUE THREAD

**Issue by acoop133** [ad-m/github-push-action#33]
Hi, can you add a parameter where it wont fail the build if nothing was commited?

Cheers

**Comment by karlito-who-else** [comment]
Is there a way of achieving this behaviour?

**Comment by ad-m** [comment]
Yes, use:
```
git commit -m "Update by GitHub Action" storage.json || echo "Nothing to update"
```
instead of:
```
git commit -m "Update by GitHub Action" storage.json || echo "Nothing to update"
```

**Comment by karlito-who-else** [comment]
Thank you for your reply @ad-m however I am struggling to see the difference between the two examples.

I *think* I managed to achieve what I needed to do using:

`git diff --quiet && git diff --staged --quiet || (git commit -m "Update by GitHub Action" -a)`

That said, if there is a simpler or more readable way to achieve this then I'll update my workflow.  I haven't tested the approach that I currently have that much yet either, so it could also be that it isn't actually working as intended.
